hbase-client/src/main/java/org/apache/hadoop/hbase/client/ColumnFamilyDescriptorBuilder.java

```diff
@@ -218,6 +221,12 @@
    */
   public static final boolean DEFAULT_BLOCKCACHE = true;
 
+  /**
+   * Default setting for whether to cache data blocks on read if block caching
+   * is enabled.
+   */
+  public static final boolean DEFAULT_CACHE_DATA_ON_READ = true;
+
```

Comment:
It's default value should be consistent with the configed BLOCKCACHE_ENABLED?

Reply:
Yeah, the BLOCKCACHE_ENABLED variable name in ColumnFamilyDescriptorBuilder is just BLOCKCACHE, which is default also true, and there the DEFAULT_CACHE_DATA_ON_READ is also true. They are the same.